LovelyCrane/Network/FireStore/UserManager.swift

```diff
@@ -9,9 +9,54 @@ import Foundation
 import Firebase
 import FirebaseFirestore
 import FirebaseFirestoreSwift
+
+struct DBUser: Codable {
```

Comment:
```suggestion
struct User: Codable {
```

DB가 꼭...있어야 할까요...호호...

Reply:
Firebase의 User 타입과 혼동할 일이 생길 수도 있을까봐 조금 다르게 네이밍 해뒀습니다!